### DISCUSSION TITLE
Error when trying to sideload

### DISCUSSION THREAD

**Discussion by piingz1337** [SoCuul/SCInsta#98] [Support] [answered]
I get this error if I want to sideload:

There was an issue during installation: 0: APIInternalError (Error Domain=IXErrorDomain Code=2 "Failed to set app extension placeholders for com.burbn.instagram.B6A2C44DWQ" UserInfo={NSLocalizedFailureReason=Failed to create promise., NSLocalizedDescription=Failed to set app extension placeholders for com.burbn.instagram.B6A2C44DWQ, FunctionName=+[IXPlaceholder _placeholderForBundle:client:withParent:installType:metadata:placeholderType:mayBeDeltaPackage:isFromSerializedPlaceholder:error:], SourceFileLine=968, NSUnderlyingError=0xacc801a70 {Error Domain=IXErrorDomain Code=8 "Attempted to set app extension placeholder promise with bundle ID com.burbn.instagram.lockscreencamera that does not match required prefix of com.burbn.instagram.B6A2C44DWQ. for parent" UserInfo={NSLocalizedDescription=Attempted to set app extension placeholder promise with bundle ID com.burbn.instagram.lockscreencamera that does not match required prefix of com.burbn.instagram.B6A2C44DWQ. for parent, FunctionName=-[IXPlaceholder setAppExtensionPlaceholderPromises:error:], SourceFileLine=1582, NSLocalizedFailureReason=Mismatched bundle IDs.}}})

I'm using iOS 18.1
@SoCuul

**Comment by SoCuul** [comment]
If I'm not wrong, it looks like you may be trying to change the bundle id (to get notifications), but that doesn't seem to be working. Which method are you using to sideload the ipa and did you make any changes after building it?

**Reply by piingz1337** [reply]
I've tried using AltStore and Sideloadly. I did not done any changes afterwards. Here are the Settings I used for Sideloadly
![Bildschirmfoto 2024-10-02 um 12 31 33](https://github.com/user-attachments/assets/11e37c67-aca8-4886-aabb-f6ec250ca21f)

**Reply by Zun4m1** [reply]
I have the same issue, it sideloads on trollstore fine though on my xr

**Reply by ArynShah** [reply]
I also have this same issue, does not work on either altstore or sideloadly but the default decrypted IPA also doesn't work so I'm not sure if this a sideloadly issue or what. I've tried IPAs from multiple sources and also tried other people's compiled IPAs, same issue.

**Reply by ic3dlem0n** [reply]
maybe its a ios 18 issue? other ipas install fine for me but this one doesnt want to work :/

**Comment by mapeles** [comment]
There seems to be a problem with the ipa we used. There was a problem when I used 350.0.0 and 351.0.0 versions, but when I made it using 334.0.0 versions of ipa, the installation was successful.

**Comment by SoCuul** [comment, marked as answer]
It should work with 350.0.0 as I have personally tested it, but your ipa could potentially be malformed in some way.
Doesn't seem like the issue is with SCInsta itself however, so I'm going to mark this as resolved.

**Reply by ic3dlem0n** [reply]
maybe its a ios 18 thing?

**Reply by SoCuul** [reply]
I'm on iOS 18.0
